api/tests/views_tests.py

```diff
@@ -125,8 +123,10 @@ def test_post_domainaddress_bad_address_error(prem_api_client) -> None:
 
     assert response.status_code == 400
     ret_data = response.json()
+    # Add unicode characters to get around Fluent.js using unicode isolation.
+    # See https://github.com/projectfluent/fluent.js/wiki/Unicode-Isolation for more info
     assert ret_data == {
-        "detail": 'Domain address "myNewAlias" could not be created, try using a different value.',
+        "detail": "“\u2068myNewAlias\u2069” could not be created. Please try again with a different mask name.",
```

Comment:
question (non-blocking): I read the fluent doc, but I don't quite understand why `"myNewAlias"` needs unicode isolation markers? Is it because of the directional quote characaters `“` and `”`? But if it is, then don't those directional quote characters needs to be *inside* the unicode isolation markers?

Reply:
I don't have a good answer for this. 

cc/ @jwhitlock

Reply:
I've got some Mozilla experience, but I don't claim to be an expert.

Other languages use [different characters for quotes](https://en.wikipedia.org/wiki/Quotation_mark#Summary_table), so I'd expect it as part of the translated pattern.

I'm not able to find guidance on quotation marks in the [styleguide](https://mozilla-l10n.github.io/styleguides/). There is an example in the [Urdu guide](https://mozilla-l10n.github.io/styleguides/ur/index.html?highlight=quotation#quotation-marks) (ur, right-to-left language):

```
Example: Instead, you might press “Enter”.
Correct: اس کے بجائے، آپ “اینٹر” دبا سکتے ہیں۔
```

I believe French would be inclined to use « » for quoting, as seen on https://github.com/mozfr/besogne/wiki/Guide-stylistique-pour-la-traduction

Reply:
@gregtatum confirmed that the isolation markers are at the right place, surrounding the value but not the quotes. This provides three benefits:

* The quote characters can be replaced, such as «»
* If the value is right-to-left and the translation is left-to-right, it is displayed correctly
* If the value is left-to-right and the translation is right-to-left, it is displayed correctly